app/assets/javascripts/search.js

```diff
@@ -0,0 +1,140 @@
+(function() {
+  "use strict";
+
+  window.GOVUK = window.GOVUK || {};
+
+  var $ = window.jQuery;
+
+  var search = {
+    init: function () {
+      var $searchResults = $('#results .results-list');
```

Comment:
Maybe scope this to `main.search` for now to avoid potential clashes with finders

Reply:
Extracted to separate CSS file so it is not included on non-site search pages